kivy/uix/behaviors/touchripple.py

```diff
@@ -138,6 +138,7 @@ def __init__(self, **kwargs):
         self.ripple_pane = CanvasBase()
         self.canvas.add(self.ripple_pane)
         self.bind(
+            disabled=lambda widget, value: widget.ripple_fade() if value else None,
```

Comment:
Minor: Instead of using a lambda, can we move this logic into a method?

Reply:
Okay, thank you!

Reply:
Updated the PR. Should I change the `def ripple_fade(self)` to `def ripple_fade(self, *args)` instead of adding the `def _ripple_fade(self, *args)`?